profiles/repo_name

```diff
@@ -1 +1 @@
-gentoo
+github
```

Comment:
Sorry! That was accidentally :-(

Reply:
bman have fixed it: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=c16c03690d690014a16ca1cc835e402ae7f941c6

Reply:
I am bman.  I caught it after committing :(